src/content/docs/en/community-resources/content.mdx

```diff
@@ -24,6 +24,7 @@ Check out the following courses and tutorials to learn more about Astro.
   - [Astro 3.0 Crash Course](https://www.youtube.com/watch?v=qBOz6TpYAOg) by James Q Quick (full paid course: [Build Modern Websites with Astro](https://astrocourse.dev/))
   - [Astro Crash Course in 60 minutes](https://www.youtube.com/watch?v=NniT0vKyn-E) by @developedbyed
   - [Understanding Astro (ebook)](https://www.ohansemmanuel.com/books/understanding-astro) by Ohans Emmanuel
+  - [The Valley of Code - Astro](https://thevalleyofcode.com/#astro) by Flavio Copes
```

Comment:
Wouldn't it be better to link to the tutorial page instead of the header on the home page that links to the tutorial?

```suggestion
  - [The Valley of Code - Astro](https://thevalleyofcode.com/astro) by Flavio Copes
```

Reply:
D'oh! This is why even *I* need an LGTM for tiny PR's!  ;)

Thank you @Fryuni!